### PR TITLE
Fix links in the doc file.

### PR DESCRIPTION
**Pull request by Yury-Fridlyand**
Signed-off-by: Yury Fridlyand <yuryf@bitquilltech.com>

### Description
Typo hotfix for PR #665.
[LinkChecker task passed](https://github.com/opensearch-project/sql/actions/runs/2658218458) for PR #665 even there were errors. Perhaps it is a bug in `LinkChecker`, but now it fails and blocks all other PRs.
 
### Issues Resolved
LinkChecker GHA to pass for new PRs.
 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by joshuali925 — DISMISSED** [review]
thanks

File: bi-connectors/PowerBIConnector/power_bi_support.md [hunk 1]
```diff
@@ -4,8 +4,8 @@
 * Microsoft Power BI Desktop
 * [OpenSearch](https://docs-beta.opensearch.org/opensearch/install/index/)
 * [OpenSearch SQL ODBC driver](https://docs-beta.opensearch.org/search-plugins/sql/odbc/)
-* [OpenSearch.mez](../../../bi-connectors/PowerBIConnector/bin/Release)
-* Optional: [sqlodbc_import.pbids](../../../bi-connectors/PowerBIConnector/PBIDSExamples) to help with repeated connections to the same server 
+* [OpenSearchProject.mez](OpenSearchProject.mez) or [AmazonOpenSearchService.mez](AmazonOpenSearchService.mez)
```
Comment (acarbonetto) [inline bi-connectors/PowerBIConnector/power_bi_support.md:7]: In the SETUP section below, it references the OpenSearch.mez file that should be updated (and make sure the folder mentioned is correct).

Comment (Yury-Fridlyand) [reply bi-connectors/PowerBIConnector/power_bi_support.md:7]: Thanks, fixed in 76313580.

**Review by acarbonetto — APPROVED** [review]

**Review by penghuo — APPROVED** [review]
LGTM

**Review by joshuali925 — APPROVED** [review]

**Pull request merged**